src/System.Diagnostics.Process/src/System/Diagnostics/AsyncStreamReader.cs

```diff
@@ -40,10 +39,8 @@ internal sealed class AsyncStreamReader : IDisposable
         private readonly Process _process;
 
         // Delegate to call user function.
-        private Action<string> _userCallBack;
+        private readonly Action<string> _userCallBack;
 
-        // Internal Cancel operation
-        private bool _cancelOperation;
         private ManualResetEvent _eofEvent;
```

Comment:
Consider changing this to be a ManualResetEventSlim.  With the usage patterns here, it won't allocate a ManualResetEvent such that the impact of not disposing the event will be minimized. In general an MRES is lighter weight than an MRE, hence the "Slim".